### DISCUSSION TITLE
Everything works, but range is too short.

### DISCUSSION THREAD

**Discussion by kaylorchen** [svpcom/wfb-ng#215] [Questions]
Hi,
I install this deb into my device, and Drone is X86, and gs is RPi 4B.
Wireless card is AC56, (8812au), I can receive video now, but max distance is 300m, and mavlink link is often lost, while video still works.
I install the 8812 driver with dkms and get it via https://github.com/svpcom/rtl8812au, branch is v5.6.4.2.
and I use "ethtool" to find dirver info:

![e4cebeadfb44a47251d3353bdf06261](https://user-images.githubusercontent.com/10471905/165423238-ba90099b-0ba1-4e84-8789-fbe2c0e5eca8.jpg)
![e4cebeadfb44a47251d3353bdf06261](https://user-images.githubusercontent.com/10471905/165423357-29d6c1e0-4f3b-4da4-82ca-237f6c9bb7c4.jpg)

drone(x86) have a empty version info, and gs(rpi) have a version info.
Is this a bug?

**Comment by kaylorchen** [comment]
![1de50fefcb12074b7299c87f767ecf7](https://user-images.githubusercontent.com/10471905/165423889-eb5c2db1-6bde-4686-8397-f87be4812fe4.jpg)
the Drone(x86) ethtools info

**Comment by svpcom** [comment]
Don't use v5.6.4.2 it has broken power control. Use v5.2.20.  See note in https://github.com/svpcom/wifibroadcast/wiki/Setup-HOWTO

**Reply by svpcom** [reply]
I've update v5.2.20 to be compatible with 5.15 kernels

**Reply by kaylorchen** [reply]
Thank you very much. I will try to change this driver.

**Reply by kaylorchen** [reply]
> 

Hello, I have update the driver and test it. Now the distance is 1.3km. I want to konw your the longest distance with RTL8812AU( AC56 wireless card). I think 1.3km is still too short.
thank you.
